site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,106 @@
 title: 'Automatic annotation'
 linkTitle: 'Automatic annotation'
 weight: 16
-description: 'Guide to using the automatic annotation of tasks.'
+description: 'Automatic annotation of tasks'
 ---
 
-Automatic Annotation is used for creating preliminary annotations.
-To use Automatic Annotation you need a DL model that can be deployed by a CVAT administrator.
-You can find the list of available models in the `Models` section.
+Automatic Annotation in CVAT is a tool that you can use
+to automatically pre-annotate your data with pre-trained models.
 
-1. To launch automatic annotation, you should open the dashboard and find a task which you want to annotate.
-   Then click the `Actions` button and choose option `Automatic Annotation` from the dropdown menu.
+You can also **add models** from popular platforms
+like **Hugging Face** or **Roboflow** to expand the
+range of available options.
+
+> Note: Automatic Annotation is a [paid CVAT Cloud feature](https://www.cvat.ai/pricing/cloud).
```

Comment:
I think this page conflates automatic annotation on self-hosted instances and on CVAT Cloud, which have very different properties:

* On self-hosted instances it's free, but you have to add your own models (using Nuclio). Roboflow / Hugging Face are not supported.
* On CVAT Cloud it is indeed a paid feature, there are several models available by default, and you can add more using Roboflow / Hugging Face.

When you talk about a property specific to a given version of CVAT, you need to make that clear.